app/models/invoice.rb

```diff
@@ -1,3 +1,9 @@
 class Invoice < ApplicationRecord
-  enum status: { 'in progress' => 0, 'completed' => 1, 'cancelled' => 2 }
+  belongs_to :customer
+  has_many :transactions
+  has_many :invoiceitems
```

Comment:
I think we'll want these to be invoice_items

Reply:
Just updated it